pkg/mapper/mapper_test.go

```diff
@@ -518,6 +608,538 @@ mappings:
 	}
 }
 
+func duplicateRules(count int, template string) string {
+	rules := ""
+	for i := 0; i < count; i++ {
+		rules += fmt.Sprintf(template, i)
+	}
+	return rules
+}
+
+func TestPerformance(t *testing.T) {
```

Comment:
Please write this in terms of a [Go benchmark](https://golang.org/pkg/testing/#hdr-Benchmarks) – it handles the number of iterations automatically for us. Also move it to a separate file, just because this one is so terribly long already.